### ISSUE TITLE
Continuous Intensity Values

### ISSUE THREAD

**Issue by anqif** [nadeemlab/SPT#155]
Currently, the data frame for each lesion contains binary values based on some cutoff. I would like to obtain the original continuous intensity values for each channel. To save memory/processing time, this should be an optional argument passed into the feature extractor function.

**Comment by jimmymathews** [comment]
Yes, this is true in the dataframe provided by the accessor `FeatureMatrixExtractor`. The data storage schema does have continuous intensity values present, after #125 was completed, so the scope of this issue is limited to the implementation of `FeatureMatrixExtractor`.

**Comment by jimmymathews** [comment]
This is worked in in branch `issue125`.
The new usage is exemplifed below.

```py
from spatialprofilingtoolbox.db.feature_matrix_extractor import FeatureMatrixExtractor
bundle = FeatureMatrixExtractor.extract('database_config_file.config', study='Melanoma intralesional IL2', specimen='lesion 6_1', continuous_also=True)
dataset = bundle['Melanoma intralesional IL2']
df = dataset['feature matrices']['lesion 6_1']['continuous dataframe']
print(df)
```

```txt
       F0        F1       F2      F3 ...
 2.285004  0.337349 19.41030 12.5641 ...
 2.870057  0.333333  7.70270 11.6577 ...
 4.986926  0.535714  7.46392 10.2474 ...
 ...
```

**Comment by jimmymathews** [comment]
This is done. Available in `spatialprofilingtoolbox==0.13.6`.
